lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
     module_function
 
     def path(dash_writes_to_stdout: false)
+      env_bundle_file = ENV.fetch(ENV_KEY, "")
+
       if ARGV.include?("--global")
+        raise "'#{ENV_KEY}' can not be specified with '--global'" unless env_bundle_file.empty?
```

Comment:
Again: use `.blank?` please.

Reply:
`.blank?` doesn't work.

```
ENV["HOME"].blank?
NoMethodError: undefined method `blank?' for "/Users/aram":String
	from (irb):2
	from /usr/bin/irb:11:in `<main>'
```

Reply:
Use `brew irb` instead. It's a ActiveSupport method included by Homebrew.

Reply:
`.blank?` works when I run `brew irb` but tests fail when I run `bundle exec rake`.